syntax = "proto3";
package pairpair.pair;


service PairService {
    rpc SavePairsHistory(Pairs) returns (Pairs);
    rpc NextPairsByHistory(Members) returns (PossiblePairs);
}


message Member {
    string name =1;
}

message Members {
    repeated Member members = 1;
}

message Pair {
    Member first = 1;
    Member second = 2;
}

message Pairs {
    repeated Pair pairs = 1;
}

message PossiblePairs {
    repeated Pairs pairs_list = 1;
}
